docs/extensibility/boot_hooks.md

```diff
@@ -53,4 +53,4 @@ we've started!
 ...snip...
 ```
 
-You have access to anything defined in the run control script environment, see [Shell Script API](https://hexdocs.pm/distillery/shell-script-api.html) for details.
+You have access to anything defined in the run control script environment, see [Shell Script API](https://hexdocs.pm/distillery/extensibility/shell_scripts.html) for details.
```

Comment:
Can you change these links to be `shell_scripts.md` instead of the full HexDocs link? The docs builder will resolve the actual url links when the docs are generated. Thanks for the PR!

Reply:
OK, I will